chore(refactor codebase ): refactored codebase based on feedback

idowuadeleke opened the pull request:
#### What does this PR do?
Implements feedback gotten from LFA to the codebase

#### Description of Task to be completed?
- change status code from 422 to 400 and from 201 to 200
- Onclick banka image should redirect back to index page
- Replace account type on the index page to Role
- Console log listening port
- The sidebar should be raised on mobile view
- Remove commented code
- Add a line at end of page

#### How should this be manually tested?
User Interface can be tested y visiting the link below
https://idowuadeleke.github.io/BANKA/

Running app with npm run start to check console logged port

Using postman to test status code

#### What are the relevant pivotal tracker stories?
#165336755

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56147627-90304780-5fa0-11e9-8f5e-f0de8bb275ea.png)
![image](https://user-images.githubusercontent.com/40268693/56147701-a8a06200-5fa0-11e9-85e6-86685800db4c.png)

Commit:
chore(refactor codebase ): rearrange codebase

this feature was used to refactor codebase and add more comments

[Delivers #165336755]

Mrshinezee commented on server/test/usertest.js:
```diff
@@ -212,7 +212,7 @@ describe('Test user login and signup', () => {
+    //         expect(body.status).to.be.equals(400);
     //         expect(body.errors.isAdmin).to.be.equals('IsAdmin field is required');
     //         done();
     //       });
```
can u remove commented code

idowuadeleke replied:
Okay

idowuadeleke replied:
This has been implemented

Mrshinezee commented on UI/CSS/style.css:
```diff
@@ -1266,5 +1242,5 @@ resize: none;
 .profiletransheader{
-  text-align:center;
+  text-align:center !important;
   border-radius:3px 3px 0 0 ;}
```
add a line at the end of the page

idowuadeleke replied:
This has been implemented

Commit:
chore(commented code ): remove all commented code

this feature was used to remove all commented code

[Delivers #165336755]

Commit:
chore(commented code ): remove all commented code

this feature was used to remove all commented code

[Delivers #165336755]

Commit:
chore(add line at end of page): add a line to end of page

this feature was used to add a line to the end of every page

[Delivers #165336755]
